Feat/kits 2.0: Enabled Multi-Industry Multi-Dataspace KITs

matbmoser opened the pull request:
## Description
<!-- 
Please describe your PR: 
- What does this PR introduce? 
- Does it fix a bug? 
- Does it add a new feature?
- Is it enhancing documentation?
-->


> [!IMPORTANT]
> Due to the many lines of code which were added (mostly because of all KIT icons were added) a Eclipse IP Check was done for this PR: https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/24651 and is APPROVED.
> Regarding the dataspace logos, it was communicated with the responsible persons for the dataspaces, to include the logos for marketing purposes, the copyright and location of the logos are at the websites from the respective dataspaces (including Catena-X), we are just using the link pointing to the respective webpage (which is under the control of the dataspace in question and can be removed at anytime)

This PR is also an example, from how can things be worked directly on open source by second dataspace organization contributing to Eclipse Tractus-X.

As described on this issue https://github.com/eclipse-tractusx/sig-release/issues/1226, the KITs are getting enabled as multi dataspace.

I am working on the base structure to refactor the structure and provide a way other dataspaces can extend their KITs and create new ones without stepping over each other. 

The current demo is available here: https://matbmoser.github.io/eclipse-tractusx.github.io/Kits
<img width="2041" height="1028" alt="image" src="https://github.com/user-attachments/assets/5e50657d-b5bc-4e9f-a89c-390c3ca528fe" />

<img width="1264" height="865" alt="image" src="https://github.com/user-attachments/assets/cb0f5a8d-682a-44f3-916e-29a37e805afe" />

<img width="2026" height="887" alt="image" src="https://github.com/user-attachments/assets/78d136a9-b6b2-4baa-abb7-cca33b78ea42" />


Additionally I am restructuring the gallery of the KITs, so that the information can be accessible easily: 
<img width="1663" height="935" alt="image" src="https://github.com/user-attachments/assets/20e318ec-36b8-4a21-ac21-85fb156b7131" />

<img width="2023" height="589" alt="image" src="https://github.com/user-attachments/assets/ece87397-b182-4de5-b957-a46f8ca29e5c" />

## Cross Dataspace Use Cases:

<img width="1911" height="947" alt="image" src="https://github.com/user-attachments/assets/ac354275-2c50-46bf-8991-7cf6c4ce7618" />

<img width="1449" height="888" alt="image" src="https://github.com/user-attachments/assets/747bb5bc-08a5-41b4-ae7e-96a647b9f4bd" />

## Industry Core Foundation

<img width="1661" height="958" alt="image" src="https://github.com/user-attachments/assets/83384a66-2a83-4cb5-ad1e-bed21d9d5247" />
<img width="1663" height="959" alt="image" src="https://github.com/user-attachments/assets/36a07b53-ec7e-4908-9713-02001bc5d722" />

## Dataspace Foundation

<img width="1666" height="952" alt="image" src="https://github.com/user-attachments/assets/11b796fd-fd24-475e-93fa-e566ce02d8eb" />
<img width="1664" height="955" alt="image" src="https://github.com/user-attachments/assets/ce740c28-6519-4f1e-9fe6-53746a681f96" />


## Deprecated Sign:

<img width="334" height="435" alt="image" src="https://github.com/user-attachments/assets/4876f10d-5bbe-462f-858e-778f03a16a10" />

## Information and New Sign

<img width="375" height="348" alt="image" src="https://github.com/user-attachments/assets/b2af38bd-cac1-4014-bddf-f6dbcd48fb53" />

<img width="238" height="312" alt="image" src="https://github.com/user-attachments/assets/b843fba7-c757-4ad6-bd49-ff49473d15a8" />


## How to contribute guide:

<img width="1795" height="1032" alt="image" src="https://github.com/user-attachments/assets/d3e79e84-907a-4e27-8d91-0ea0272a3306" />
<img width="1799" height="830" alt="image" src="https://github.com/user-attachments/assets/548a43f4-2d3b-4556-8174-7eed8f425eb3" />
<img width="1802" height="642" alt="image" src="https://github.com/user-attachments/assets/13b5406e-9b11-43f5-8e42-af4c7b59a212" />
<img width="1802" height="423" alt="image" src="https://github.com/user-attachments/assets/ef1d8490-5c97-45ff-8a08-90fd1fa6f9a2" />

## KIT Lifecycle

<img width="1053" height="966" alt="image" src="https://github.com/user-attachments/assets/32aed2df-40b8-4444-86eb-aa306f177296" />


## KIT Master Data Overview

<img width="1036" height="959" alt="image" src="https://github.com/user-attachments/assets/d1899429-d472-44a7-8d03-306d1d71d94f" />
<img width="1075" height="830" alt="image" src="https://github.com/user-attachments/assets/fd0c36b8-6476-4a74-8b35-5e4bead6557a" />


## KIT Statistics
<img width="1166" height="930" alt="image" src="https://github.com/user-attachments/assets/aa555d04-9d59-4a14-afe1-6812955aa128" />

<img width="1020" height="824" alt="image" src="https://github.com/user-attachments/assets/6a5cf983-c0a6-44b6-9d3a-e3078ac45c70" />
<img width="1102" height="598" alt="image" src="https://github.com/user-attachments/assets/fbc3749c-98b5-4f98-ac97-5d229bc7ee0b" />
<img width="1065" height="945" alt="image" src="https://github.com/user-attachments/assets/568306c3-3f7b-4142-8f57-52b99808c1b1" />

## Deprecation Process

<img width="1048" height="557" alt="image" src="https://github.com/user-attachments/assets/78a262cb-a082-446d-9c71-ec77e62498b1" />


## KIT Framework
<img width="1062" height="949" alt="image" src="https://github.com/user-attachments/assets/03bb7592-438b-411e-b660-4ccc8db4d6c2" />

## KIT Sub Navigation

<img width="406" height="369" alt="image" src="https://github.com/user-attachments/assets/2ad9682a-d00c-4bf7-bd92-f7584a52b08c" />



## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

Closes https://github.com/eclipse-tractusx/sig-release/issues/1226

matbmoser commented on src/components/AboutUsContent/index.js:
```diff
@@ -34,7 +34,7 @@ export default function AboutUsContent() {
 
           <p className="description-p">
-            The Eclipse Tractus-X™ project is the official open-source project in the Catena-X ecosystem under the umbrella of the Eclipse Foundation. The Eclipse Foundation is not-for-profit corporation that it supported by over 320 members, and represents the worlds largest sponsored collection of Open Source projects and developers.
+            The Eclipse Tractus-X™ project is the official open-source project in the Catena-X ecosystem and several other Manufacturing-X ecosystems under the umbrella of the Eclipse Foundation. The Eclipse Foundation is not-for-profit corporation that it supported by over 320 members, and represents the worlds largest sponsored collection of Open Source projects and developers.
```
fair and done

matbmoser commented:
I added a disclaimer for every dataspae

CDiezRodriguez commented on utils/generated/README.md:
NOTICE License?

CDiezRodriguez reviewed (changes requested):
Amazing work, @matbmoser!

I tested this locally, and everything launched and worked perfectly. I also reviewed the webpage, and it looks great.

I've left a few minor comments on the files, but otherwise, this looks good to go.

I only have a few points:

1. There's one link I wasn't sure about—I don't know if it's meant to redirect to a valid page, so I left a comment.
2. There were a few other links that seemed to be broken. I assumed their intended destination and adjusted them to match.
3. Finally, I found a file without the license. I also have some doubts about the CHANGELOG file, which I've pointed out as well.

matbmoser commented:
Just to be sure I am checking one by one the kits to see if the version is aligned with the master data.

matbmoser commented:
Ok there were several kits which were wrong dated

matbmoser commented:
I needed to add a short update, otherwise always when someone builds the website it will update

matbmoser commented:
there were some typos in the statistics

matbmoser commented:
I added a devider on the kits architecture

matbmoser commented:
Any other change I will include in the next PR: https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/1378

matbmoser commented:
The disclaimer text was not fully complete

matbmoser commented:
And also as a note for the future, I have moved the draft TRG 09-01 which was for the KITs and split it into 3 new 10.xx TRGs which were made active with this update (so I applied to the website some rules) when you create KITs we need to do some changes in the master data and keep some quality parameters which are described in this TRGs.

We need still to discuss how we graduate a KIT or how we deprecated it, but for that I have created two more TRGs in draft.
